shared/test/otSdkDriver_test.js

```diff
@@ -157,6 +157,26 @@ describe("loop.OTSdkDriver", function() {
         sinon.match.instanceOf(HTMLDivElement),
         expectedConfig);
     });
+
+    it("should call onPublishComplete", function() {
```

Comment:
Please can you wrap these two in a describe, e.g. `describe("initPublisher complete", {...`